ui/src/artist/ArtistExternalLink.js

```diff
@@ -36,9 +36,29 @@ const ArtistExternalLinks = ({ artistInfo, record }) => {
     linkButtons.push(<span key={`link-${record.id}-${id}`}>{link}</span>)
   }
 
-  addLink(links[0], 'message.openIn.lastfm', <ImLastfm2 />)
+  const musicBrainzIcon = (
+    <>
+      <svg width="0" height="0" xmlns="http://www.w3.org/2000/svg">
+        <defs>
+          <linearGradient id="mb-icon-gradient" x2="1">
+            <stop stopColor="var(--mb-icon-color-one, #FFF)" offset="0%" />
+            <stop stopColor="var(--mb-icon-color-one, #FFF)" offset="50%" />
+            <stop stopColor="var(--mb-icon-color-two, #FFF)" offset="50%" />
+            <stop stopColor="var(--mb-icon-color-two, #FFF)" offset="100%" />
+          </linearGradient>
+        </defs>
+      </svg>
```

Comment:
What is this svg required for? If it is really required for the functionality proposed, can't we just move it to the `MusicBrainzIcon` file itself?

Reply:
@srichter, did you see this comment?

Reply:
The gradient lets you style the MusicBrainz icon with two colors (to match the official logo). It does make more sense to be inside the svg itself; moved it there.